### PR TITLE
Title & description update

### PR DESCRIPTION
**Pull request by silenzushka**
Changed title in the manifest for better store visibility.

### CODE REVIEW

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "appName": {
-    "message": "Behave!",
+    "message": "Behave! — Offensive Comments Blocker",
```
Comment (iislucas) [inline app/_locales/en/messages.json]: Rather than "Blocker" (because it lets you see them) maybe it is an offensive comments  "controller"?

Comment (silenzushka) [inline app/_locales/en/messages.json]: Totally agree that controller suites the case better, but people search for blockers (i.g. ad blockers, comments blockers, and they allow to "unblock" too). This title change meant to help the app get more organic users.

Comment (ftroitsky) [inline app/_locales/en/messages.json]: @iislucas Think the goal for this change is to make the app more accessible to the wider audience. "Controler" sounds like a complex thing right away. "Filter" maybe?

Comment (iislucas) [inline app/_locales/en/messages.json]: Got it. Then best to give a description of what it does for you: "Behave! - control the toxic  comments you see.". or "Behave! - choose your threshold for offensive comments.". The words "filter" and "block" are a little misleading. Thoughts?

Comment (ftroitsky) [inline app/_locales/en/messages.json]: But "Blocker" is a simple wording. Less geeky. And I think Alex has done some research on it for better store visibility, so I am OK with it.

Comment (ftroitsky) [inline app/_locales/en/messages.json]: Got your point, will think a bit.

Comment (silenzushka) [inline app/_locales/en/messages.json]: @iislucas description is provided right after the title in the store. The title is used to get more relevant users when they search for specific terms.

Comment (ftroitsky) [inline app/_locales/en/messages.json]: The full picture:

**Category**
Productivity

**Keywords**
- block, 
- comments,
- toxic,
- youtube,
- Reddit,
- offensive comments

**Title**
Behave! — Offensive Comments Blocker

**Short Title**
Behave!

**Tag Line**
Hides comments on YouTube, Reddit & Twitter depending on the desired level of toxicity and converts them into harmless emojis.

**Description**
Tired of offensive comments on your favorite social networks and media sites? Behave! allows you to set a level of toxicity you are willing to tolerate and replaces comments that exceed it with graphical emojis.

You are able to unhide any comment and read it. Don’t like it? Hide it again. You are in control now.

Click the Behave! icon in the Chrome toolbar and use a slider to pick the toxicity level according to your current mood.

Behave! works on:
YouTube
Reddit
Twitter

English is the only supported language at the moment.

Comment (silenzushka) [inline app/_locales/en/messages.json]: Lucas, we can't make title longer, as there is a 45 characters limit which will not fit your suggestions. I suggest we brainstorm later and most likely come up with something better.

Comment (ftroitsky) [inline app/_locales/en/messages.json]: 👍

Comment (iislucas) [inline app/_locales/en/messages.json]: I see, and you already have a longer description.... two more (shoter) ideas... 
"Behave! - offensive comments masked by emoji" or 
"Behave! - toxic comments hidden by emoji"

Comment (silenzushka) [inline app/_locales/en/messages.json]: Thanks, @iislucas! These are very good. I'll split test them during coming 2 weeks and see which one converts better.

**Pull request merged**